packages/babel-parser/src/parser/location.js

```diff
@@ -98,6 +98,11 @@ export const Errors = Object.freeze({
   MissingUnicodeEscape: "Expecting Unicode escape sequence \\uXXXX",
   MixingCoalesceWithLogical:
     "Nullish coalescing operator(??) requires parens when mixing with logical operators",
+  ModuleAttributesIncorrectValue:
+    "Only string literals are allowed as module attributes values",
+  ModuleAttributesWithDuplicateKeys:
+    "Module attributes should not include duplicate keys (%0)",
```

Comment:
```suggestion
    "Duplicate key \"%0\" is not allowed in module attributes",
```

Reply:
This definitely is better. Made the changes 👍